docs/app/e2e/app.scenario.js

```diff
@@ -39,7 +39,7 @@ describe('docs.angularjs.org', function () {
 
 
     it('should change the page content when clicking a link to a service', function () {
-      browser.get('build/docs/index.html');
+      browser.get('build/docs/index-production.html');
```

Comment:
Yes, so there are other tests in this folder that hit the normal `index.html`.
Also all the main e2e tests from the examples are run against local angular.